### PR TITLE
feat: repo README updates

### PR DESCRIPTION
**Pull request by merklefruit**
Updated most READMEs in the repo.

### CODE REVIEW

**Review by thedevbirb — APPROVED** [review]
Thanks sir 🫡

File: README.md [hunk 1]
```diff
@@ -85,34 +87,45 @@ sequenceDiagram
...
   the best bid to proposers for block proposal.
 - [**MEV-Boost**](./mev-boost/): A fork of the [Flashbots MEV-Boost][fb-mev-boost] sidecar
   that includes new API endpoints to proxy requests from the Bolt Sidecar to the connected relays.
 - [**Bolt Contracts**](./bolt-contracts/): A set of smart contracts for peripheral functionality
   such as proposer registration and permissionless dispute resolution for attributable faults.
+- [**Bolt Boost**](./bolt-boost/): A [Commit-Boost][commit-boost] module that implements the
+  Constraints-API and is compatible with the Bolt Sidecar.
+- [**Bolt client**](./bolt-client/): A CLI tool to send preconfirmations on enabled test networks.
+- [**Bolt Kurtosis client**](./bolt-kurtosis-client/): A CLI tool to send preconfirmations on our Kurtosis devnet.
+- [**Bolt delegations CLI**](./bolt-delegations-cli/): A CLI tool to generate signed delegation messages for ETH validators.
```
Comment (namn-grg) [inline README.md]:
```suggestion
- [**Bolt delegations CLI**](./bolt-delegations-cli/): A CLI tool to generate signed delegation and revocation messages for ETH validators.
```

File: README.md [hunk 2]
```diff
@@ -85,34 +87,45 @@ sequenceDiagram
...
+- [**Testnets**](./testnets/): A set of guides and scripts to deploy the Bolt contracts on testnets.
+
+Additionally, this repository contains the necessary scripts to spin up a [Kurtosis][kurtosis]
+devnet with all the components running.
 
-Additionally, this repository contains the necessary scripts to spin up a [Kurtosis][kurtosis] devnet
-with all the components running, and a simple [web demo](./bolt-web-demo/) to showcase the preconfirmation flow.
+> [!NOTE]
+> Bolt also works with external components that aren't part of this repository,
+> such as [Helix][helix] by Gattaca, which will replace the MEV-Boost relay soon.
```
Comment (namn-grg) [inline README.md:124]: should we add link to the helix fork somewhere?

**Review by namn-grg — APPROVED** [review]
thanks LGTM!

File: README.md [hunk 3]
```diff
@@ -16,44 +16,46 @@
 
 > [!IMPORTANT]
 > Bolt is an implementation of _permissionless proposer commitments through
```
Comment (mempirate) [inline README.md]: I think the language here focuses too much on PBS as a tool to achieve preconfs, while in reality we're just compatible with it. Maybe something like

> Bolt is an implementation of permissionless proposer commitments that is fully compatible with PBS. In its essence, it consists in a fork of the MEV-Boost stack that allows users to request commitments like preconfirmations from proposers, and then adds a way for proposers to commit to transaction inclusion in a way that is verifiable on-chain.

File: README.md [hunk 4]
```diff
@@ -151,20 +165,16 @@ just build-images
 
 # spin up the kurtosis devnet on your machine
 just up
-
-# run the web demo servers.
-just demo
 ```
 
 The web demo will be available on your browser at [`http://localhost:3000`](http://localhost:3000).
```
Comment (mempirate) [inline README.md]: Is it still available without `just demo`?

**Review by mempirate — CHANGES_REQUESTED** [review]
Great job, some nits

File: README.md (hunk 4 above)
Comment (merklefruit) [reply README.md:170]: sory, what is available?

File: README.md (hunk 2 above)
Comment (merklefruit) [reply README.md:124]: done!

File: README.md (hunk 3 above)
Comment (merklefruit) [reply README.md]: I like it, ty

File: README.md (hunk 4 above)
Comment (merklefruit) [reply README.md]: removed ref

**Pull request merged**